packages/dx-grid-core/src/plugins/table-edit-row/computeds.ts

```diff
@@ -27,7 +27,7 @@ export const tableRowsWithEditing: TableRowsWithEditingFn = (
     }));
 
   return [
-    ...addedTableRows.reverse(),
+    ...Array.prototype.slice.call(addedTableRows).reverse(),
```

Comment:
Can we create an util in dx-core, for instance, that will look like below:

```js
const slice = arr => Array.prototype.slice.call(arr); // slice can be renamed to copy as well

// usage
...slice(addedTableRows).reverse();
```

The reason: 'Array.prototype.slice' looks too verbose.